### PR TITLE
Swift

### PR DESCRIPTION
**Pull request by swizzlr**
Docker library PR: https://github.com/docker-library/official-images/pull/2500

### CODE REVIEW

**Comment by swizzlr** [comment]
Idiot that I am, I didn't use the content.md file. Fixing now.

**Comment by swizzlr** [comment]
HOORAY! @tianon who else needs to have eyes on this?

File: swift/content.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+docker start swiftfun
+```
+
+and then attach it
+
+```bash
+docker attach swiftfun
+```
+
+## Contributions
```
Comment (yosifkit) [inline swift/content.md]: Sections similar to your `Contributions` and `License` sections will be generated at the end of the README; similar to [the end of bash](https://github.com/docker-library/docs/tree/40720962b381977eebc676242912f793036e385d/bash#license).

Comment (swizzlr) [inline swift/content.md]: Apologies; will fix when I'm at my 💻  asap.

**Comment by swizzlr** [comment]
@yosifkit thanks for that – fixed! Please merge at your/@tianon's convenience!

**Pull request merged**